src/actions/profile-view.js

```diff
@@ -280,6 +280,8 @@ type TrackInformation = {|
   // This is the tab that should be selected from this track. `null` if this
   // track doesn't have a prefered tab.
   relatedTab: null | TabSlug,
+  // This is the initial track reference
```

Comment:
yeah exactly, I'll improve the comment.
I need this so that we can save it to the state later. I was also thinking to store a `TrackInformation` instead, but thought that `TrackInformation` is more a "temporary data exchange" structure rather than something to be stored...